apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx

```diff
@@ -117,12 +117,12 @@ as $$
 begin
   return query
   select
-    id,
-    content,
+    documents.id,
+    documents.content,
     1 - (documents.embedding <=> query_embedding) as similarity
   from documents
   where 1 - (documents.embedding <=> query_embedding) > similarity_threshold
```

Comment:
does this also need to change to `match_threshold`?

Reply:
Yes the variable name `similarity_threshold` should also be changed to `match_threshold` in the `where` clause to ensure consistency with the examples later in the post.

Reply:
would you be able to make the change @nathanrchn ? then I can merge it in! thanks 🙏

Reply:
Of course I didn't understand your comment at first, sorry for that.